### ISSUE TITLE
Accessibility: Toolbar (DashNav) buttons contrast too low 

### ISSUE THREAD

**Issue by MaximKipot** [grafana/grafana#22813]
Unfortunately, the icons in the Navbar are not accessible too

**Steps to reproduce**:
use any contrast checker of your choice

**Actual Result**:
Contrast ratio is 4.08, minimal is 4.5
<img width="456" alt="Screenshot 2020-03-16 at 15 06 53" src="https://user-images.githubusercontent.com/1902515/76769457-d1c30700-679c-11ea-9d92-6dedd4359f0f.png">

**Expected Result**
Contrast ratio is 4.97 with a color of #989898. 
As far as I understood $text-color-weak and $text-muted should be updated.
<img width="494" alt="Screenshot 2020-03-16 at 15 42 38" src="https://user-images.githubusercontent.com/1902515/76769480-d8ea1500-679c-11ea-89ee-d5d0d6782d44.png">

**Relevant WCAG Criteria:** [1.4.3 WCAG Criterion](link to https://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-contrast)

**Environment**:
- Grafana version: 6.7.0-pre
- Data source type & version: TestData DB coming with Simple dummy streaming example
- User OS & Browser: MacOS Mojave, Mozilla and Chrome
